node/src/consensus/babe_consensus.rs

```diff
@@ -200,6 +219,7 @@ impl ConsensusMechanism for BabeConsensus {
         _task_manager: &mut TaskManager,
         _client: Arc<FullClient>,
         _triggered: Option<Arc<std::sync::atomic::AtomicBool>>,
```

Comment:
There are at least three names for the same system-wide trigger. Please, unify them.

Reply:
Ah I see now, good idea let me do that 👍🏼